lib/pacio_test_kit/common_tests/adi_search_tests/patient_search_tests/patient_deathdate_family_search_test.rb

```diff
@@ -0,0 +1,34 @@
+require_relative '../../../searcht_test'
+
+module PacioTestKit
+  class PatientDeathdateFamilySearchTest < Inferno::Test
+    include PacioTestKit::SearchTest
+
+    title 'Server returns valid results for search by deathdate + family'
+    description %(
+      A server SHALL support searching by deathdate + family.
+      This test will pass if resources are returned and match the search criteria. If
+      none are returned, the test is skipped.
+    )
+
+    id :patient_deathdate_family_search_test
+
+    def tag
+      config.options[:profile]
+    end
+
+    def self.properties
+      @properties ||= SearchTestProperties.new(
+        resource_type: config.options[:resource_type],
+        search_params: [
+          { name: 'death-date', paths: ['death-date'] },
```

Comment:
the path for [`death-date`](https://www.hl7.org/fhir/us/core/SearchParameter-us-core-patient-death-date.html) is `deceased`.